port/unix/omrsysinfo.c

```diff
@@ -1325,7 +1325,28 @@ omrsysinfo_get_s390_zos_supports_transactional_execution_facility(void)
 	uint8_t CVTFLAG4 = *(CVT + 0x17B);
 
 	/* Note we check for both constrained and non-constrained transaction support */
```

Comment:
We should update this comment to reflect the split check for constrained and non-contraction transactional memory